### PR TITLE
Fix All Recent CVEs

### PR DESCRIPTION
**Pull request by sumansreenivasan**
### JIRA link ###

https://tools.hmcts.net/jira/browse/RDM-10583

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -116,7 +116,8 @@
     "lodash.template": "^4.5.0",
     "websocket-extensions": "^0.1.4",
     "bl": "^3.0.1",
-    "dot-prop": "^4.2.1"
+    "dot-prop": "^4.2.1",
+    "ini": "^1.3.7"
```
Comment (MSancaktutar) [inline package.json:120]: would this be accompanied with a yarn lock update?

Comment (sumansreenivasan) [reply package.json:120]: added updated yarn.lock

**Review by MSancaktutar — APPROVED** [review]
LGTM

**Comment by MSancaktutar** [comment]
Merged with confirmation with Ola.

**Pull request merged**